Behavioral.Automation.Bindings.UI.Tests/Configurations/WebElementStorageService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Behavioral.Automation.Bindings.UI.Abstractions;
+
+namespace Behavioral.Automation.Bindings.UI.Tests.Configurations;
+
+public class WebElementStorageService : IWebElementStorageService
+{
+    public T Get<T>(string locatorKey)
+    {
+        IWebElement webElement = new WebElement();
+        return (T) webElement;
```

Comment:
And what is going to happen if `webElement` could not be cast to `T`?

Reply:
The transformation here is used only for testing. In BDD.Implementation.Playwright we can add meaningful error message. I save your comment for the next PR.

Reply:
ok, got it!

Reply:
Added where T: IWebElementWrapper to interface declaration